app/assets/javascripts/discourse/app/templates/components/user-fields/text.hbs

```diff
@@ -1,9 +1,9 @@
-<label class="control-label" for={{concat "user-" this.elementId}}>
-  {{this.field.name}}
-  {{#if this.field.required}}<span class="required">*</span>{{/if}}
-</label>
 <div class="controls">
   <Input id={{concat "user-" this.elementId}} @value={{this.value}} maxlength={{this.site.user_field_max_length}} />
+  <label class="control-label" for={{concat "user-" this.elementId}}>
```

Comment:
This also affects the "Create Account" form, we should make sure it doesn't regress there.

Reply:
I've tested there and looks fine.